_pages/en/mandate.md

```diff
@@ -8,40 +8,52 @@ sections: Mission
+- Challenge current thinking and approaches
 
-The [Canadian Experience](human-development-life-cycle.html) is when Canadians are benefiting seamlessly of the federal government services through modern technologies.
+<img src="assets/images/our-values.png" alt="">
+- Learning and Continuous Improvement
+- Self-Discipline
+- Innovation
+- Collaboration
+- Communication
+- Teamwork
 
-Continuous improvement transformation and GC wide "Government as a service" based on [GC Digital Standards](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html) and the [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601). See [Medium Term IT Picture - 2025](it-picture-medium-term.html) and [Long Term IT Picture - 2030](it-picture-long-term.html).
+<img src="assets/images/our-mission.png" alt="">
+Enable the strategic value of IT within ESDC by reducing its risks and increasing business agility.
 
-## Mission (team's reason for existence)
+<img src="assets/images/our-goals.png" alt="">
+- Develop transformation growth strategies for the organization
+- Help the organization to adopt  DevOps that dramatically reduce risks associated with technology investments and  unlock significant service improvement capabilities
+- Deliver value very quickly, using very short lead times
+- Speed up the time to organization, apply incremental improvements in response to the changing environment, and create a more streamlined development process
+- Continuously iterate and innovate 
 
-Enable the strategic value of IT within ESDC by reducing its risks and increasing business agility.
+<img src="assets/images/our-vision.png" alt="">
+Our vision is to release changes of all kinds on demand quickly, safely, and sustainably so that [Canadians are benefiting seamlessly](human-development-life-cycle.html) of the federal government services through modern technologies.
+Continuous improvement transformation and GC wide "Government as a service" based on [GC Digital Standards](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html) and the [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601). See [Medium Term IT Picture - 2025](it-picture-medium-term.html) and [Long Term IT Picture - 2030](it-picture-long-term.html).
 
-## Mandate (team's authority within IITB)
 
-- Diagnose key organizational problems, waste, lead times and governance structures that inhibit the desired changes in behaviours
-- Recommend specific course of actions to governance bodies to adopt new methods of managing IT risks
-- Measure progress of adoption of strategies in IT teams and governance bodies as a performance indicator of the IT Strategy team
-- Challenge current thinking and approaches
 
-## Team's Guiding Principles
+### **Team's Guiding Principles**
```

Comment:
There is some overlap between this content on guiding principles and the content above on 'Our Values'. Can we condense into one section?

Reply:
Also wondering if this content (team values/principles) would be better on the About us page rather than the Mandate page? Just a thought.

Reply:
What I really like about the principles is that we explain what the principle means for our team. If I was to combine the values list and the principles list, I would keep the format and content of the principles list and add in:

"Teamwork - we swarm to solve problems and respond to requests, we have our own files but we also work across files sharing our expertise to help one another" or something like that and 

"Working in the open - we strongly believe in sharing our work openly on this website. We include here not just final products but also products in development so that we can collaborate and improve our work and also so that our work can be re-used by anyone." or something like that.

Reply:
ok, I will leave the format of "Team principles" and will add those two points. Thanks ;)